src/components/Photo/DocumentLiveCapture.js

```diff
@@ -62,7 +62,7 @@ export default class DocumentLiveCapture extends Component<Props, State> {
   captureDocumentPhoto = () => {
     this.setState({ isLoading: true })
     sendEvent('Taking live photo of document')
-    screenshot(this.webcam, this.captureDocument)
+    screenshot(this.webcam, this.captureDocument, 'image/jpeg')
```

Comment:
How does this compare with the upload of a png? You should be able to see the upload time on Woopra

Reply:
Looks promising from Woopra stats during a quick check. Duration taken is definitely less with JPEG compared to the PNG upload.